awx/ui/src/screens/InstanceGroup/InstanceGroupEdit/InstanceGroupEdit.test.js

```diff
@@ -70,27 +67,6 @@ describe('<InstanceGroupEdit>', () => {
     jest.clearAllMocks();
   });
 
-  test('controlplane instance group name can not be updated', async () => {
```

Comment:
This seems like a good use case for an integration test to confirm that the api throws a validation error when trying to change this @tiagodread @akus062381 @InnocentK